install/kubernetes/helm/helm-service-account.yaml

```diff
@@ -1,7 +1,7 @@
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Please revert back to v1 as of 1.8 RBAC is GA

Reply:
If I use /v1, it doesn't work with k8s 1.7.

Reply:
BTW, the rest of istio is also using v1beta1 (see the rbac-beta yaml file)

Reply:
well, if you have to support 1.7 then I guess there is no other solution.

Reply:
yes, that is what istio currently supports from istio.io.  btw, this change is already in master, I didn't separate my new change out of this one cleanly.